cypress/config/vault_config.json

```diff
@@ -4,9 +4,7 @@
       "secret": "Crossroads.Automation",
       "keys": [
         "CONTENTFUL_SPACE_ID", 
-        "CONTENTFUL_ACCESS_TOKEN", 
-        "BITMOVIN_BUCKET", 
```

Comment:
BITMOVIN_BUCKET and BITMOVIN_DIRECTORY were do not exist in the Crossroads.Automation secret. I also looked in Netlify, Github actions as well to see if I could find them. I decided to remove them since they are not being populated from vault.